packages/components/src/components/checklist/checklist.tsx

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
+import React, { MouseEventHandler } from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import Icon from '../icon';
 import Button from '../button';
 
-const ItemStatus = ({ status, onClick, button_text }) => {
+type TItemStatusProps = {
+    button_text: string;
+    onClick: MouseEventHandler<HTMLButtonElement | HTMLDivElement>;
```

Comment:
The better approach for giving type to our onClick would be () => void;
We mostly use React.MouseEvent whenever we get event as one of our parameters.  (e: React.MouseEvent<HTMLElement> ) => void

Reply:
didn't know the difference. thanks @hirad-rewok . changed it.

Reply:
@hirad-rewok checked it and seems like we don't need `MouseEventHandler<HTMLButtonElement | HTMLDivElement>`. updated the type. please check it again :pray: